examples/list_view_apps_launcher/application_row/imp.rs

```diff
@@ -0,0 +1,35 @@
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use gtk::{gio, glib};
+
+use gtk::CompositeTemplate;
```

Comment:
I think we can drop this, it's actually re-exported by gtk::subclass::prelude::*;

Reply:
It is needed :)